public/app/views/index.html

```diff
@@ -163,6 +163,43 @@ <h5 style="margin-top: 20px; margin-bottom: 40px"><strong>GO client</strong></h5
                 <h5 style="margin-top: 20px; margin-bottom: 40px"><strong>Swift/iOS client</strong></h5>
                 <!--<p class="text-muted">Each Bootstrap theme includes all the source and compiled files you'll need, including npm and Gulp based build tools.</p>-->
             </div>
+
+            <div class="col-sm-2">
+                <a href="https://github.com/OpenSocket/socketcluster-client-ruby">
+                    <img class="homepage-grid-icon" src="https://image.flaticon.com/icons/svg/919/919842.svg"
+                         style="width: 25%; height: 25%">
+                </a>
+                <h5 style="margin-top: 20px; margin-bottom: 40px"><strong>Ruby client (Beta) </strong></h5>
+                <!--<p class="text-muted">Each Bootstrap theme includes all the source and compiled files you'll need, including npm and Gulp based build tools.</p>-->
```

Comment:
Why did you include all those HTML comments? :slightly_smiling_face:

Reply:
Ok I will remove it....